sdk/eventhub/event-hubs/src/linkEntity.ts

```diff
@@ -126,9 +131,16 @@ export class LinkEntity {
       this.name,
       this.address
     );
-    await defaultLock.acquire(this._context.cbsSession.cbsLock, () => {
-      return this._context.cbsSession.init();
-    });
+    await defaultCancellableLock.acquire(
+      this._context.cbsSession.cbsLock,
+      () => {
+        return this._context.cbsSession.init({ abortSignal });
+      },
+      {
+        abortSignal,
+        acquireTimeoutInMs: getRetryAttemptTimeoutInMs(undefined)
```

Comment:
Shouldnt the caller be sending the timeout value? We are assuming 60 seconds here.

Reply:
When called in the loop for token renewal, we can keep the default 60 sec, but other calls are coming from the sender, receiver and the mgmt link, all of which has a timeout that they can pass

Reply:
The token renewal will keep the 60 seconds, but now the caller will send the timeout value here.